src/main/python/image_preprocessing/face_manager.py

```diff
@@ -25,6 +25,30 @@ def __init__(self, shape_predictor_path):
         """
         self.shape_predictor_path = shape_predictor_path
 
+    def face_likeness_extraction(self, image):
+        """
+        This function find a face in the image passed and is optimised
+        to align the face before being returned.
+        Author(s):
+            Stephan Nell
+        Args:
+            image (:obj:'OpenCV image'): Image containing the face we need to detect and extract.
+        Raises:
+            ValueError: If no face can be detected.
+        Returns:
+            obj:'OpenCV image': Any background and unnecessary components are removed and only
+            the aligned face is returned in gray scale for facial likeness.
+        Todo:
+            Return error if no face detected
+
+        """
+        rectangle = self.detect(image)
+        predictor = dlib.shape_predictor(self.shape_predictor_path)
+        face_aligner = FaceAligner(predictor, desiredFaceWidth=256)
+        face_aligned = face_aligner.align(image, image, rectangle)
+        gray_face = cv2.cvtColor(face_aligned, cv2.COLOR_BGR2GRAY)
```

Comment:
No, don't return a tuple or a packed value. These things should only be returned when absolutely necessary and not be misused. I have looked at `extract_face` and seen that the only extra steps are that it returns a copy of the image passed (?) as part of a packed value (?). I will, however, merge this. When we do a code cleanup, we will discuss this again (as well as the `extract_face` function.